packages/agent/app/index.html

```diff
@@ -19,6 +42,9 @@
   </head>
 
   <body>
+    <div id="splash">
+      <img src="./assets/bigtest-logo.svg">
```

Comment:
😍 😍 😍 

If the agent is going to just be standing around, might as well be wearing nice clothes, right? love to see it...